Framework/Subset_of_CorLib/System/DateTime.cs

```diff
@@ -93,13 +88,18 @@ public struct DateTime
         private const ulong TickMask = 0x7FFFFFFFFFFFFFFFL;
         private const ulong UTCMask = 0x8000000000000000L;
 
-        public static readonly DateTime MinValue = new DateTime(MinTicks);
-        public static readonly DateTime MaxValue = new DateTime(MaxTicks);
+        // Ticks at 12:00 AM January 1, year 1601 A.D.

```

Comment:
The native part was designed based on 12:00 AM January 1, year 1601 A.D. as the epoch. Technically, it was also possible to fix it from the native layer but required more changes with a greater amount of regression risk. So we decided to fix it from the managed layer and leave the native part untouched (at least for this version).